src/EventStore.Core/Services/Transport/Http/Authentication/ClientCertificateAuthenticationProvider.cs

```diff
@@ -20,25 +21,34 @@ public class ClientCertificateAuthenticationProvider : IHttpAuthenticationProvid
 			if (clientCertificate is null) return false;
 
 			bool hasReservedNodeCN;
+			string clientCertificateCN;
 			try {
-				hasReservedNodeCN = clientCertificate.GetNameInfo(X509NameType.SimpleName, false) == _certificateReservedNodeCommonName;
+				clientCertificateCN = clientCertificate.GetCommonName();
+				hasReservedNodeCN = clientCertificateCN == _certificateReservedNodeCommonName;
 			} catch (CryptographicException) {
 				return false;
 			} catch (NullReferenceException) {
 				return false;
 			}
 
+			if (!hasReservedNodeCN) {
+				var ip = context.Connection.RemoteIpAddress?.ToString() ?? "<unknown>";
+				Log.Error(
+					"Connection from node: {ip} was denied because it's CN: {clientCertificateCN} does not match with the reserved node CN: {reservedNodeCN}",
```

Comment:
```suggestion
					"Connection from node: {ip} was denied because its CN: {clientCertificateCN} does not match with the reserved node CN: {reservedNodeCN}",
```